threshold_policy_enforcer.go

```diff
@@ -131,7 +132,9 @@ type evaluationNode struct {
 	// by rule ID.
 	childNodes map[int][]*evaluationNode
 
-	// commited indicates whether the node is commited or not.
+	// commited indicates whether more child (virtual) nodes will be added to
+	// this node. If set to true, the node is finalized and no more child
```

Comment:
Is `finalized` confusing? Even if it is committed, the node can still be undetermined. And the `finalized` function may still return false?

```go
// A node is finalized if either it's determined or any of its ancestors is
// determined.
func (n *evaluationNode) finalized() bool {
	node := n
	for node != nil {
		if node.ruleDecision != thresholdPolicyDecisionUndetermined {
			return true
		}
		node = node.subjectNode
	}
	return false
}
```

Reply:
good call, finalized is different from commited, I removed the confusing part.